packages/fast-usdc/src/types.ts

```diff
@@ -62,9 +62,13 @@ export interface PendingTx extends CctpTxEvidence {
 }
 
 export type FeeConfig = {
+  /** flat fee charged for every advance */
   flat: Amount<'nat'>;
+  /** percentage charged for every advance */
   variableRate: Ratio;
-  maxVariable: Amount<'nat'>;
+  /** a cap on variable fees. no cap if undefined */
```

Comment:
I read this at first as a limit on how the `variableRate` could move. Now I understand it as a max on the variable component of the total fee.

Why have a max on that component instead of the fee overall? They're functionally equivalent offset by `flat`, but having a `maxFee` is much clearer to the user.